README.md

```diff
@@ -87,5 +87,5 @@ campaign_placement = "mtm_placement"
 ```
 
 For example, by default parameter `campaign_name` track following parameters if they are found in URL: `'mtm_campaign', 'matomo_campaign', 'mtm_cpn', 'utm_campaign'`. If you configure `campaign_name` like this `campaign_name="mtm_campaign,custom_name_parameter"`, then parameter `campaign_name` will detect only presence of `mtm_campaign` and `custom_name_parameter` in URL. `matomo_campaign`, `mtm_cpn`, `utm_campaign` will be ignored until they are present in config.
-All parameter are case-insensitiv except optional mtm_clid.
+All parameter values are case‑sensitive. For example, if you set `campaign_group = "mtm_group"`, it will match only `mtm_group`, not `MTM_GROUP`.
```

Comment:
I'm not sure if this is right. I believe that PG-4454 was regarding the utm_id not being recorded when the URL used `&utm_ID=` instead of `&utm_id=`. The parameter _key_ is always case-sensitive, but the _value_ is not unless the `Keep campaign parameter capitalisation` setting is enabled.

Reply:
@AltamashShaikh I just realised that I was mixing things up somewhat. I was thinking of the URL parameter key/value while the FAQ is in the context of mapping multiple parameter keys to the same internal key. Your initial text was good given that context. We probably don't need to add the bit about the values. How about something like this?
 ```
 All parameter keys are case‑sensitive. For example, if you set `campaign_group = "mtm_group"`, it will match only `mtm_group`, not `MTM_GROUP`, `mtm_GROUP`, etc.
 ```

Reply:
Actually, your last commit looks good. :+1: